src/microbe_stage/MicrobeAI.cs

```diff
@@ -223,6 +223,17 @@ private void ChooseActions(Random random, MicrobeAICommonData data, Microbe? sig
                 break;
         }
 
+        // If I'm very far from the player, get on stage
+        var player = data.AllMicrobes.Where(otherMicrobe => otherMicrobe.IsPlayerMicrobe).FirstOrDefault();
```

Comment:
Do we want to lump this in with the spawn system change which is already a huge difference?

Reply:
I'm pretty confident this is now essential. With microbes now spawning in a ring far from the player, without this change you would need to spawn almost 5 times as many to get the same number to appear on screen.

Reply:
Wasn't the idea with sector spawns that sectors contain specific amount of stuff, which would even load in and out as the player moves? Did this PR fundamentally change from the first spawn system PR?

Reply:
I'm not intimately familiar with the other PR, but based on what you're describing this is a much simpler solution. All this change does is take the "Biased circle" spawn pattern and instead track what areas have and have not spawned something in them. It then relies on existing despawn functionality to remove things too far from the player.

This works fine for chunks and clouds which don't move, but most microbes won't just sit around and wait to be discovered; they will start moving, usually out of the perimeter for depsawning and never enter the game properly. My suspicion is that this is already an issue in master, but now that the Microbes are spawning a little further out (so they don't appear onscreen) it's a more noticeable problem.

Reply:
Ah okay, you should rename this PR to not contain the "sector" word in the name then.